packs/equipment/linnorms-sankeit.json

```diff
@@ -12,7 +12,7 @@
+            "value": "<p>The first suit of <em>+3 greater resilient antimagic sankeit</em> made in this fashion was crafted for the Varki linnorm king Nankou, after he claimed his title by slaying a taiga linnorm. By Varki tradition, the armor was crafted using some of the slain linnorm's body to decorate the breastplate and helm, which imbued the armor with several of the linnorm's natural abilities. Though the helm is shaped like the beast's head, a linnorm's actual head would be too large for a proper helmet.</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">2</span> Interact</p>\n<p><strong>Frequency</strong> once per hour</p><hr /><p><strong>Effect</strong> You breathe a @Template[cone|distance:60] of electrified vapor, dealing @Damage[17d6[electricity]|options:area-damage] to creatures in the area (@Check[reflex|dc:38|basic|options:area-effect]). The electrified mist persists in the area for 2 rounds, dealing @Damage[5d6[electricity]|options:area-damage] (DC 38 basic Reflex save) to each creature that ends its turn in the mist. If you slew the linnorm this armor is made from, you can use the higher of your class DC or spell DC instead of the listed DCs for this effect (@Check[reflex|against:class-spell|basic]).</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">r</span> Interact</p>\n<p><strong>Trigger</strong> A creature adjacent to you targets you with a melee attack</p><hr /><p><strong>Effect</strong> The creature takes @Damage[4d6[piercing]] as previously invisible magical spines leap outwards from the armor to punish them for the attack.</p><hr /><p><strong>Craft Requirements</strong> The initial raw materials must include the hide and skull of a taiga linnorm.</p>"
```

Comment:
```suggestion
            "value": "<p>The first suit of <em>+3 greater resilient antimagic sankeit</em> made in this fashion was crafted for the Varki linnorm king Nankou, after he claimed his title by slaying a taiga linnorm. By Varki tradition, the armor was crafted using some of the slain linnorm's body to decorate the breastplate and helm, which imbued the armor with several of the linnorm's natural abilities. Though the helm is shaped like the beast's head, a linnorm's actual head would be too large for a proper helmet.</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">2</span> Interact</p>\n<p><strong>Frequency</strong> once per hour</p><hr /><p><strong>Effect</strong> You breathe a @Template[cone|distance:60] of electrified vapor, dealing @Damage[17d6[electricity]|options:area-damage] to creatures in the area (@Check[reflex|dc:38|basic|options:area-effect]). The electrified mist persists in the area for 2 rounds, dealing @Damage[5d6[electricity]|options:area-damage] (DC 38 basic Reflex save) to each creature that ends its turn in the mist. If you slew the linnorm this armor is made from, you can use the higher of your class DC or spell DC instead of the listed DCs for this effect (@Check[reflex|against:class-spell|basic|options:area-effect]).</p><hr /><p><strong>Activate</strong> <span class=\"action-glyph\">r</span> Interact</p>\n<p><strong>Trigger</strong> A creature adjacent to you targets you with a melee attack</p><hr /><p><strong>Effect</strong> The creature takes @Damage[4d6[piercing]] as previously invisible magical spines leap outwards from the armor to punish them for the attack.</p><hr /><p><strong>Craft Requirements</strong> The initial raw materials must include the hide and skull of a taiga linnorm.</p>"
```